warehouse/upload_test.go

```diff
@@ -381,6 +381,7 @@ func TestUploadJobT_UpdateTableSchema(t *testing.T) {
 						AlertSender: &mockAlertSender{
 							mockError: tc.mockAlertError,
 						},
+						AlterEnabled: true,
```

Comment:
We could have the attribute be called `DisableAlter` instead so that the default (i.e. `false`) would make more sense.